### DISCUSSION TITLE
How to run pyodide with vue.js locally without internet connection?

### DISCUSSION THREAD

**Discussion by larspava** [pyodide/pyodide#4295] [Q&A] [answered]
I want to use pyodide inside  my vue.js that runs inside an android application. One of the requirements is that it should run offline, in the vue 2 version. I tried to use pyodide with "npm install pyodide" command and by downloading the package from github, so it could run the files locally, but didn't get any success.

Is it possible to use the package locally without internet connection?

Below is my last trying using [this tutorial](https://pyodide.org/en/0.19.1/usage/index.html#node-js): 
Here's the packages from the project:
![image](https://github.com/pyodide/pyodide/assets/63216415/d49de735-1baf-42f6-bb3a-81e19e2ea47c)

Code snipet:
!![image](https://github.com/pyodide/pyodide/assets/63216415/a346ff6d-0dd6-42c7-b760-df65b8414f8b)

Error messages:
![image](https://github.com/pyodide/pyodide/assets/63216415/1c68b3f4-68bc-43d9-a6e0-fcc49cbb0532)

Directory estructure:
├───public
│   └─── index.html
└───src
    ├───components
    │   ├───home
    │   	└───HomeView.vue    
    ├───pyodide
    │   └───fonts
    │   └───fonts
    │   └───fonts
    ├───views
    └───App.vue
    package.json
    package-lock.json

**Comment by ryanking13** [comment, marked as answer]
Running a vue.js application on Android sounds like you're saying you want to launch a browser environment with a webview and run Pyodide inside of it. (I don't know much about Android development or app development, so please correct me if I'm wrong).

In that case, since you cannot access local file system in the browser, I think you need to 1) download the [Pyodide distribution](https://github.com/pyodide/pyodide/releases/tag/0.24.1), 2) run a local server that serves it, and 3) set the indexURL to point that local server.

**Reply by larspava** [reply]
Yes, that's exactly what I want.
With your tip I get to run the server locally. But, since the app launch a browser environment, it could not get access to the pyodide files so there's no much I can do in this scenario. After some reaserch, we get it to run using a Python SDK for Android.
Thanks for the answer.
